official/nlp/finetuning/binary_helper.py

```diff
@@ -308,6 +308,53 @@ def write_glue_classification(task,
       else:
         # Classification.
         writer.write('%d\t%s\n' % (index, class_names[prediction]))
+def write_superglue_classification(task,
+                                   model,
+                                   input_file,
+                                   output_file,
+                                   predict_batch_size,
+                                   seq_length,
+                                   class_names,
+                                   label_type='int',
+                                   min_float_value=None,
```

Comment:
Seems like `min_float_value` and `max_float_value` are not used? 
(I think SuperGlue has no regression task so it's safe to remove them anyway?